### ISSUE TITLE
Elasticsearch::Config type error?

### ISSUE THREAD

**Issue by siliconalchemy** [elastic/puppet-elasticsearch#740]
* Module version: 0.15.0
* Puppet version: 4.7.0
* OS and version: CentOS 7

## Bug description
```
Error: Could not set 'present' on ensure: no implicit conversion of Fixnum into String at 131:/srv/puppet/environments/production/modules/elasticsearch/manifests/config.pp
Error: Could not set 'present' on ensure: no implicit conversion of Fixnum into String at 131:/srv/puppet/environments/production/modules/elasticsearch/manifests/config.pp
Wrapped exception:
no implicit conversion of Fixnum into String
Error: /Stage[main]/Elasticsearch::Config/Sysctl[vm.max_map_count]/ensure: change from absent to present failed: Could not set 'present' on ensure: no implicit conversion of Fixnum into String at 131:/srv/puppet/environments/production/modules/elasticsearch/manifests/config.pp
```

In manifests/config.php, changing line 131 from:
 value => 262144,
to
 value => "262144",
fixes the problem.

**Comment by tylerjl** [comment]
Interesting; must be a version-specific issue. Changing to a string seems harmless however; so I'll PR a fix for the next bugfix release. Thanks for the report, @siliconalchemy!

**Comment by tylerjl** [comment]
Merged in #741, will be in the next release.

**Comment by siliconalchemy** [comment]
Thanks :)
